lib/game.rb

```diff
@@ -69,6 +69,13 @@ def play
       puts @player_2.board.render(true)
       @player_2.dead?
     end
+      if @player_2.dead?
+        puts ('=' * 40)
+        puts "You suck"
+      else @player_1.dead?
```

Comment:
I don't think the else statement needs the "@player_1.dead?". Because it should be the only other option available after @player_2.dead?